riff-raff/app/AppComponents.scala

```diff
@@ -108,7 +108,8 @@ class AppComponents(
 
       Map(
         "gu:build-tool" -> buildTool.getOrElse(default),
-        "gu:repo" -> repoUrl.getOrElse(default)
+        "gu:repo" -> repoUrl.getOrElse(default),
+        "gu:riff-raff-project" -> projectName
```

Comment:
minor: Should we namespace this as `gu:riff-raff:project`? This'll be similar behaviour to the [GuCDK tags](https://github.com/guardian/cdk/blob/main/src/constants/metadata-keys.ts). Saying that, the `gu:build-tool` and `gu:build-identifier` are not namespaced 🤷🏽‍♂️.